include/soci/postgresql/soci-postgresql.h

```diff
@@ -309,6 +309,11 @@ struct postgresql_statement_backend : details::statement_backend
 
     typedef std::map<std::string, char **> UseByNameBuffersMap;
     UseByNameBuffersMap useByNameBuffers_;
+
+    // the following map is used to keep the results of column
+    // type queries with custom types
+    typedef std::map<unsigned long, char> CategoryByColumnOID;
```

Comment:
Not a big deal, but should use `unordered_map` here IMO -- not only it's (marginally, for the maps of this size) more efficient, but it's also more clear as we don't really need this map to be sorted.

BTW, the reason for using `map` above is that this code had been written before SOCI could use C++11, but this is not the case any longer.

Reply:
Yes my thought was exactly to use the same type to avoid doing it differently, but i agree with unordered_map being better.